frontend/src/components/regions/UserActions/ActivityLog.jsx

```diff
@@ -41,26 +39,27 @@ export default function ActivityLog({ eventList, sortedTimestampArray }) {
   return (
     <React.Fragment>
       {sortedTimestampArray.map((timestamp) => (
```

Comment:
The name of this array could be better. We did this as an example but if i were to look at it now, I'm thinking this array is an array of timestamps that have been sorted and nothing else.